ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/state/Fork.java

```diff
@@ -20,7 +20,9 @@
 
 public class Fork {
 
+  // TODO previous_version should be a bytes4 (this has serialization impacts)
   private Bytes previous_version;
+  // TODO previous_version should be a bytes4 (this has serialization impacts)
```

Comment:
should this be current_version?

Reply:
I think this implementation is actually correct https://github.com/ethereum/eth2.0-specs/blob/v0.5.1/specs/core/0_beacon-chain.md#fork

Reply:
I think @cemozerr is referring to the name on the comment @dangerousfood, it should be current_version

Reply:
84784e1